var/spack/repos/builtin/packages/fzf/package.py

```diff
@@ -25,8 +28,24 @@ class Fzf(MakefilePackage):
 
     depends_on('go@1.11:')
 
+    variant('vim', default=False, description='Install vim plugins for fzf')
+
+    patch("github_mirrors.patch")
+
     def edit(self, spec, prefix):
+        # the ${HOME}/.glide directory must exist on some systems
```

Comment:
@clellsolomon Do you know why this is needed? If possible it would be great to avoid this: generally Spack package installations don't write outside of the Spack root directory, so this might defy some folks' expectations.